mappings/net/minecraft/world/gen/placementmodifier/RangeDecoratorConfig.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2997 net/minecraft/world/gen/decorator/RangeDecoratorConfig
+CLASS net/minecraft/class_2997 net/minecraft/world/gen/placementmodifier/RangeDecoratorConfig
```

Comment:
This is actually a `FeatureConfig` (a config used to make `ConfiguredFeature` from `Feature`). Should probably be `net.minecraft.world.gen.RangeFeatureConfig`.